styles/styles.css

```diff
@@ -576,6 +588,303 @@ navigation-item-main {
   fill: rgba(117, 97, 87, 0.3);
 }
 
+.modal-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(242, 242, 242, 0.8);
+}
+
+/* .modal-container-close {
+  display: none;
+} */
+
+.modal-search {
+  margin: auto;
+  background-color: #ffffff;
+  width: 715px;
+  border-radius: 30px;
+}
+
+.modal-content {
+  margin: 62px 70px 70px;
+  position: relative;
+}
+
+.modal-title {
+  margin: 0;
+  position: relative;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.modal-close-button {
+  position: absolute;
+  padding: 0;
+  top: -10px;
+  right: 0;
+  width: 53px;
+  height: 53px;
+  background-color: #f2f2f2;
+  border: none;
+  border-radius: 50%;
+
+  cursor: pointer;
+}
+
+.button-close {
+  fill: #000000;
+}
+
+.modal-close-button:hover {
+  background: #e6e6e6;
+}
+
+.modal-close-button:focus {
+  border: 3px solid #83b3d3;
+  background: #e6e6e6;
+}
+
+.modal-close-button:active .button-close {
+  opacity: 0.3;
+}
+
+.modal-close-button:active {
+  background: #e6e6e6;
+}
+
+.field-group {
+  position: relative;
+  display: flex;
+  margin: 0;
+  margin-top: 40px;
+  justify-content: space-between;
+}
+
+.people {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 40px;
+}
+
+.form-name {
+  padding-top: 15px;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 26px;
+  text-transform: capitalize;
+}
+
+.field {
+  font-size: 18px;
+  line-height: 21px;
+  background-color: #f2f2f2;
+  border: 0;
+  border-radius: 4px;
+}
+
+.date-form-field {
+  position: relative;
+  padding: 15px 60px 14px 19px;
+  width: 341px;
+}
+
+.date-form-field:hover {
+  background: #e6e6e6;
+}
+
+.date-form-field:focus {
+  box-shadow: inset 0 0 0 3px #83b3d3;
+  background: #e6e6e6;
+}
+
+.date-form-field:active {
+  background: #ffffff;
+  box-shadow: inset 0 0 0 1px #000000;
+}
+
+.button-modal-form {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  border: 0;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-color: transparent;
+  cursor: pointer;
+  opacity: 0.5;
+}
+
+.calendar-button {
+  top: 0;
+  right: 0;
+  background-image: url("/images/calendar.svg");
```

Comment:
```suggestion
  background-image: url("../images/calendar.svg");
```
Так будет правильно. Иначе браузер файл изображения не находит. Форма выглядит вот так:
![image](https://user-images.githubusercontent.com/17790771/178107656-dfe161ec-7c08-4798-b545-645ffd5df9f7.png)